src/doc/rustc/src/platform-support/armv6k-nintendo-3ds.md

```diff
@@ -0,0 +1,104 @@
+# `armv6k-nintendo-3ds`
+
+**Tier: 3**
+
+The Nintendo 3DS platform, which has an ARMv6K processor, and its associated
+operating system (`horizon`).
+
+## Target maintainers
+
+- [@Meziu](https://github.com/Meziu)
+- [@AzureMarker](https://github.com/AzureMarker)
+- [@ian-h-chamberlain](https://github.com/ian-h-chamberlain)
+
+## Requirements
+
+This target is cross-compiled. `#![no_std]` crates should work without much
+additional effort.
```

Comment:
Oops, I just realized this is probably not completely true, do `no_std` crates still need to use `-Zbuild-std=core` or something like that?

Reply:
Yeah I think they need core to be built via build-std.

Reply:
Plus, the C RunTime is available only in devkitARM, so that’s another thing.

(Now that I think about it, mention that the platform doesn’t support dynamic linking).